spec/uploaders/avatar_uploader_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'carrierwave/test/matchers'
+
+describe AvatarUploader do
+  include CarrierWave::Test::Matchers
+
+  let(:user)      { create(:user) }
+  let(:uploader)  { AvatarUploader.new(user, :avatar) }
+
+  before do
+    AvatarUploader.enable_processing = true
+    File.open('spec/assets/default-avatar.png') { |f| uploader.store!(f) }
+  end
+
+  after do
+    AvatarUploader.enable_processing = false
+    uploader.remove!
+  end
+
+  context 'the thumb version' do
+    it 'scales down avatar to be exactly 64 by 64 pixels' do
+      expect(uploader.thumb).to have_dimensions(64, 64)
```

Comment:
I think resize_to_fit preserves the original aspect ratio, so the resulting image could have lower dimensions for either width or height.